packages/zoe/src/contracts/multipoolAutoswap/types.js

```diff
@@ -43,6 +43,10 @@
  * @property {() => ZCFSeat} getPoolSeat
  * @property {() => Amount} getSecondaryAmount
  * @property {() => Amount} getCentralAmount
+ * @property {() => Notifier} getNotifier
+ * @property {() => Void} updateState
```

Comment:
```suggestion
 * @property {() => void} updateState
```

Reply:
done.  Why does typescript let me get away with this?